glotaran/cli/commands/optimize.py

```diff
@@ -32,6 +33,14 @@
     help="Path to an output directory.",
     show_default=True,
 )
+@click.option(
+    "--outformat",
+    "-ofmt",
+    default="folder",
+    type=click.Choice(known_project_formats()),
```

Comment:
This might raise a `NotImplementedError`, it should be filtered see the plugin support table generation:
https://github.com/glotaran/pyglotaran/blob/5efcf0c94da9f68c79441fe88c9e497f8860a2f9/glotaran/plugin_system/project_io_registration.py#L494